django_spanner/features.py

```diff
@@ -217,6 +217,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         "admin_views.test_multidb.MultiDatabaseTests.test_change_view",
         "admin_views.test_multidb.MultiDatabaseTests.test_delete_view",
         "auth_tests.test_admin_multidb.MultiDatabaseTests.test_add_view",
+        "auth_tests.test_remote_user_deprecation.RemoteUserCustomTest.test_configure_user_deprecation_warning",
```

Comment:
There is no such a test in the current Django version. That related to [the issue](https://github.com/googleapis/python-spanner-django/issues/485) about using an old Django fork for testing (it is 2000 behind the original Django version)